src/modules/commentUI.js

```diff
@@ -0,0 +1,39 @@
+export default class CommentUI {
+  static addFacts = async (parentElement, facts) => {
+    // clear old entries
+    parentElement.innerHTML = '';
+    // [type, shortcut, author]
+    const fact1 = document.createElement('p');
+    fact1.textContent = `Type: ${facts[0]}`;
+    const fact3 = document.createElement('p');
+    fact3.textContent = `Shortcut: ${facts[2]}`;
+    const fact4 = document.createElement('p');
+    fact4.textContent = `Author: ${facts[3]}`;
+
+    parentElement.appendChild(fact1);
+    parentElement.appendChild(fact3);
+    parentElement.appendChild(fact4);
+
+    return parentElement;
+  };
+
+  static showComments = (
+    ele,
+    { creation_date: date, comment: message, username: name },
+  ) => {
+    const {
+      creation_date: creationDate,
+      comment,
+      username,
+    } = { creation_date: date, comment: message, username: name };
+    const comEl = document.createElement('p');
+    comEl.innerText = `${creationDate} ${username} ${comment} `;
+    // console.log(comEl);
+    ele.appendChild(comEl);
+    return comEl;
+  };
```

Comment:
- [ ] The number of comments is not updated after adding a comment. Please ensure that when the user adds a comment, the number of comments changes immediately without closing the pop-up.

Reply:
Done